autoload/lightline/colorscheme/papercolor_light.vim

```diff
@@ -0,0 +1,259 @@
+" =============================================================================
+" Filename: autoload/lightline/colorscheme/autoload/lightline/colorscheme/papercolor_light.vim
+" Author: TKNGUE
+" License: MIT License
+" Last Change: 2015/07/21 13:18:03
+" =============================================================================
+
+fun! <SID>grey_number(x) "{{{
```

Comment:
Please do not repeat yourself. These utility functions for papercolor_(light|dark) can be moved to autoload/lightline/colorscheme/papercolor.vim.
```
<SID>grey_number -> function! lightline#colorscheme#papercolor#gray_number()
```
Same notice for all the other functions.